deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 packageName: ibm-block-csi-operator-community
 channels:
 - name: stable
```

Comment:
as I already [wrote](https://github.com/IBM/ibm-block-csi-operator/pull/186/files#r659147162) (in a very similar PR) - I don't think this is the correct channel for 1.5.1

Reply:
@ArbelNathan some information about OLM channels:
https://olm.operatorframework.io/docs/best-practices/channel-naming/

Reply:
Is it our decision?

Reply:
yes, we need an additional channel

Reply:
added new channel